src/client/graphics/layers/ChatModal.ts

```diff
@@ -0,0 +1,305 @@
+import { LitElement, html } from "lit";
+import { customElement, query } from "lit/decorators.js";
+
+import { GameView, PlayerView } from "../../../core/game/GameView";
+
+import quickChatData from "../../../../resources/QuickChat.json";
+import { EventBus } from "../../../core/EventBus";
+import { SendQuickChatEvent } from "../../Transport";
+
+type QuickChatPhrase = {
+  key: string;
+  text: string;
+  requiresPlayer: boolean;
+};
+
+type QuickChatPhrases = Record<string, QuickChatPhrase[]>;
+
+const quickChatPhrases: QuickChatPhrases = quickChatData;
```

Comment:
If it does not create a compile error, it would be nice to jeep the narrow type:
```suggestion
const quickChatPhrases = quickChatData satisfies QuickChatPhrases;
```